parl/__init__.py

```diff
@@ -14,5 +14,13 @@
 """
 generates new PARL python API
 """
-from parl.framework import *
+
+from parl.utils.utils import _HAS_FLUID
+from parl.utils import logger
+
+if _HAS_FLUID:
+    from parl.framework import *
+else:
```

Comment:
It's unnecessary to warn users because some users want to use parl.remote_class only.

Reply:
print("WARNING:PARL: Failed to import paddle, only APIs for parallelization are available.")